wdae/wdae/genomes_api/tests/test_gene_models_api.py

```diff
@@ -40,6 +40,11 @@ def test_get_nonexistant_gene_transcripts(anonymous_client):
     assert response
     assert response.status_code == 404
 
+def test_get_case_insensitive_gene(anonymous_client):
+    response = anonymous_client.get("/api/v3/genome/gene_models/default/cHd8")
+
+    assert response
+    assert response.status_code == 200
```

Comment:
Validate the response data is actually the CHD8 gene, not just the 200 status code